### PR TITLE
move get_to_yast.pm over to the test cases (as installation/bootloader_s390.pm

### PR DESCRIPTION
**Pull request by froh**

### CODE REVIEW

File: tests/installation/bootloader_s390.pm [hunk 1]
```diff
@@ -0,0 +1,427 @@
...
+    # line was not long enough to split it.  Then add the closing
+    # paren and the Newline now.
+    $parmfile_with_Newline_s .= "\")\nNewline"
+      unless $parmfile_with_Newline_s =~ /Newline\n$/s;
+
+    return $parmfile_with_Newline_s;
+}
+
+use backend::s390x::gnah;
+
```
Comment (coolo) [inline tests/installation/bootloader_s390.pm]: this should live in here too I assume?

File: tests/installation/bootloader_s390.pm [hunk 2]
```diff
@@ -0,0 +1,427 @@
...
+    ### say "\$sequence=@$sequence";
+
+    $self->{s3270}->sequence_3270(@$sequence);
+
+    return $r_screenshot;
+}
+
+###################################################################
+require Text::Wrap;
+
```
Comment (coolo) [inline tests/installation/bootloader_s390.pm]: it's not really part of the concept that tests require addtional non-standard modules. Perhaps you can include the wrap function ? it shouldn't be that much code

Comment (coolo) [inline tests/installation/bootloader_s390.pm]: (see the test suite failure)

Comment (froh) [inline tests/installation/bootloader_s390.pm]: I don't understand.
The warnings in the test suite come from the Test::Wrap API which works with setting variables, which is ugly but is what it is.
or are you suggesting to reimplement Text::Wrap ??

Comment (coolo) [inline tests/installation/bootloader_s390.pm]: How many of these lines do you really need?
http://cpansearch.perl.org/src/MUIR/Text-Tabs+Wrap-2013.0523/lib.modern/Text/Wrap.pm

Comment (coolo) [inline tests/installation/bootloader_s390.pm]: you don't need to reimplement it - it's GPL :)

Comment (froh) [inline tests/installation/bootloader_s390.pm]: Text::Wrap is a core module.

I'll simply split the local declaration and the assignment onto two lines.  that will shut up the warnings, like so:

    local $Text::Wrap::columns;
    $Text::Wrap::columns = ...;

Comment (coolo) [inline tests/installation/bootloader_s390.pm]: hmm, I see. The existance of perl-Text-Wrap confused me :)

**Pull request merged**